python/dlr/counter/config.py

```diff
@@ -0,0 +1,12 @@
+CALL_HOME_URL = 'https://r4sir2jo37.execute-api.us-east-2.amazonaws.com/testing/'
+CALL_HOME_RECORD_FILE = 'ccm_record.txt'
+CALL_HOME_USER_CONFIG_FILE = 'ccm_config.json'
+CALL_HOME_PUBLISH_MESSAGE_MAX_QUEUE_SIZE = 100
+CALL_HOME_MAX_WORKERS_THREADS = 5
+CALL_HOME_MODEL_RUN_COUNT_TIME_SECS = 5
+CALL_HOME_USR_NOTIFICATION = """\n CALL HOME FEATURE DISABLED 
```

Comment:
This is misleading, this sounds like it is already disabled, but it should be more like this is what you need to do to disable this feature

Reply:
The notification text has corrected and moved to setup tool script.

Reply:
After modify user notification text message and incorporate into setup utility,
Below is python installation command shows the message ..
 
/neo-ai-dlr/python$ python3 setup.py install --user

 CALL HOME FEATURE ENABLED
                            
 Your device information and model load/execution count metric data being sent to a server.                             
 You can disable this feature by using below configuration steps.                             
	1. Create a config file, ccm_config.json in your application path.                             
	2. Added below format content in it,                             
		{
			"ccm" : "false"
		}                             
	3. Restart DLR application.
running install
running bdist_egg
running egg_info
...